kano_init_flow/ui/main_window.py

```diff
@@ -89,6 +90,15 @@ def _key_emergency_exit(self, widget, event):
 
         return False
 
+    def _key_skip_stage(self, widget, event):
+        if hasattr(event, 'keyval'):
+            if event.keyval in [Gdk.KEY_N, Gdk.KEY_n] and \
+               event.state & Gdk.ModifierType.SHIFT_MASK and \
+               event.state & Gdk.ModifierType.CONTROL_MASK:
```

Comment:
This is quite an `if` statement - with the least change, this can be jus one if right?

```python
if hasattr(event, 'keyval') and \
        event.keyval in [Gdk.KEY_N, Gdk.KEY_n] and \
        event.state & Gdk.ModifierType.SHIFT_MASK and \
        event.state & Gdk.ModifierType.CONTROL_MASK:
```

It just feels like this can be written better.